AutomatedTesting/Gem/PythonTests/AWS/Windows/aws_metrics/aws_metrics_automation_test.py

```diff
@@ -3,24 +3,38 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
+
+
+Prerequisites for running AWS automation tests on your local Windows machine:
+1) Go to the AWS IAM console and create an IAM role called o3de-automation-tests which adds your own AWS account as
+   a trusted entity and has the admin permissions

```

Comment:
minor: "... an IAM role called o3de-automation-tests which adds your own account as as a trusted entity and uses the "AdministratorAccess" permissions policy"

Reply:
Sure. I'll make the change.